lib/wordmove/cli.rb

```diff
@@ -58,6 +58,23 @@ def logger
       end
     end
 
+    desc "list", "List all environments and vhosts"
+    shared_options.each do |option, args|
+      method_option option, args
+    end
+    def list
+      Wordmove::List.start(options)
+    rescue Wordmove::MovefileNotFound => e
+      puts e.message
+      exit 1
+    rescue Psych::SyntaxError => e
+      puts "Your movefile is not parsable due to a syntax error: #{e.message}"
+      exit 1
+    rescue LoadError, StandardError => e
+      puts "Your movefile is not parsable. Please confirmt its format: #{e.message}"
```

Comment:
I wonder how to deal other critical and unexpected errors except  MovefileNotFound or SyntaxError.
If LoadError is caught, the message like ``puts "Your movefile is not parsable. Please confirmt its format: #{e.message}"`` is not suitable.

I think replace the message or not to handle the other errors is the better.
I'm going to remove this ``LoadError, StandardError`` block.

Reply:
Agree. Keep in mind that exceptions handling in this software is implemented at its minimum. If we had a more complete and mature exception handling infrastructure it'd been easier. Since we have not, it turns out of the scope of the PR